learn/faq.md

```diff
@@ -18,6 +51,33 @@ For more information, please check out the [conda documentation page on channels
 
 [Conda-forge](https://conda-forge.org/) is a [community-led GitHub organization](https://github.com/conda-forge) that provides access to thousands of conda package recipes. All of these recipes are open source and can be installed with the conda package manager by specifying conda-forge as the channel.
```

Comment:
I know this isn't your addition, Pinak, but we should be sure we're consistent with the conda-forge naming.

@jaimergp, is it supposed to always be lowercase, or is capitalizing at the beginning of a sentence fine?